tests/items/src/varargs.c

```diff
@@ -80,6 +80,16 @@ void valist_struct_member(const char *fmt, ...) {
   va_end(thestruct.args);
 }
 
+// pattern first seen in graphviz (sftable.c)
+void valist_struct_pointer_member(const char *fmt, ...) {
+  struct vastruct thestruct;
+  struct vastruct *pointer = &thestruct;
+
+  va_start(pointer->args, fmt);
+  vprintf(fmt, pointer->args);
+  va_end(pointer->args);
```

Comment:
It's not testing `va_copy`.  That might be handled similarly to `va_start` and `va_end`, but I think it should still be tested as well.  Can you add that?  Might as well add the same to the similar `valist_struct_member` function above.